src/Service/Relationships.php

```diff
@@ -218,12 +218,30 @@ private function normalizeRelationships(array $relationships)
         return $relationships;
     }
 
+    /**
+     * Returns whether the database supports MariaDB command names (added in MariaDB 10.4.6).
+     *
+     * See: https://jira.mariadb.org/browse/MDEV-21303
+     *
+     * @param array $database The database definition from the relationships.
+     * @return bool
+     */
+    public function supportsMariaDBCommands(array $database)
+    {
+        if (isset($database['type']) && (strpos($database['type'], 'mariadb:') === 0 || strpos($database['type'], 'mysql:') === 0)) {
+            list(, $version) = explode(':', $database['type'], 2);
+            return version_compare($version, '10.5', '>=');
```

Comment:
this is the server version rather than the client version

but perhaps it's a good enough heuristic